### ISSUE TITLE
ES_PLUGINS_INSTALL is not working

### ISSUE THREAD

**Issue by pySilver** [wodby/elasticsearch#1]
Setting `ES_PLUGINS_INSTALLS` as per doc: `module1,module2` causes error (nothing is installed. Single module installs just fine. Im not a shell expert but my bet the issue is there.

**Comment by pySilver** [comment]
This are the logs when >1 module is mentioned in env var:

```
Attaching to foo_elasticsearch_1
elasticsearch_1  | A tool for managing installed elasticsearch plugins
elasticsearch_1  |
elasticsearch_1  | Commands
elasticsearch_1  | --------
elasticsearch_1  | list - Lists installed elasticsearch plugins
elasticsearch_1  | install - Install a plugin
elasticsearch_1  | remove - removes a plugin from Elasticsearch
elasticsearch_1  |
elasticsearch_1  | Non-option arguments:
elasticsearch_1  | command
elasticsearch_1  |
elasticsearch_1  | Option         Description
elasticsearch_1  | ------         -----------
elasticsearch_1  | -h, --help     show help
elasticsearch_1  | -s, --silent   show minimal output
elasticsearch_1  | -v, --verbose  show verbose output
elasticsearch_1  | ERROR: Found multiple arguments for option [arguments], but you asked for only one
foo_elasticsearch_1 exited with code 64
```

**Comment by csandanov** [comment]
Thank you for reporting this, fixed in 4.2.0
